CRM/Contact/Form/Task/PDFTrait.php

```diff
@@ -505,7 +505,7 @@ public function formatMessage(&$message) {
       ],
     ];
 
-    $htmlMsg = preg_split($newLineOperators['p']['pattern'], $message);
+    $htmlMsg = preg_split($newLineOperators['p']['pattern'], ($message ?? ''));
```

Comment:
Is there a unit test where this is coming up or was this from a mass find/replace? Since $message is from the form values html message field or an uploaded doc, being null would be an error condition, so the error message is useful here. Having said that, it would have been silent before, and I suppose the person would notice their pdf is blank.

Reply:
@demeritcowboy there are all coming up from my running unit tests locally on php8.1

Reply:
Do you remember which test is passing in null here? It makes me think either the test is faulty or, well, I am faulty.